addons/web/static/tests/views/form/form_view_tests.js

```diff
@@ -6700,29 +6700,26 @@ QUnit.module("Views", (hooks) => {
         await click(form.el.querySelector(".o_field_x2many_list_row_add a"));
     });
 
-    QUnit.skipWOWL("modifiers are considered on multiple <footer/> tags", async function (assert) {
-        assert.expect(2);
-
-        const form = await makeView({
+    QUnit.test("modifiers are considered on multiple <footer/> tags", async function (assert) {
+        await makeView({
```

Comment:
Change the test to create a Webclient and call an inline action with target="new"